packages/language-server/src/completion/completions.json

```diff
@@ -80,6 +80,10 @@
       "label": "url",
       "documentation": "Connection URL including authentication info. Each datasource provider documents the URL syntax. Most providers use the syntax provided by the database [learn more](https://pris.ly/d/connection-strings)."
     },
+    {
+      "label": "directUrl",
```

Comment:
Can we update this to use the `insertText` per the relationMode completion and the `fullSignature`? Something that shows how we expect directUrl to be used